packages/rum/src/domain/record/serialize.spec.ts

```diff
@@ -114,22 +115,41 @@ describe('serializeNodeWithId', () => {
         style: 'width: 10px;',
       })
     })
-
-    it('serializes scroll position', () => {
-      const element = document.createElement('div')
-      Object.assign(element.style, { width: '100px', height: '100px', overflow: 'scroll' })
-      const inner = document.createElement('div')
-      Object.assign(inner.style, { width: '200px', height: '200px' })
-      element.appendChild(inner)
-      sandbox.appendChild(element)
-      element.scrollBy(10, 20)
-
-      expect((serializeNodeWithId(element, DEFAULT_OPTIONS)! as ElementNode).attributes).toEqual(
-        jasmine.objectContaining({
+    ;[
+      {
+        description: 'serializes scroll position during full snapshot',
+        serializationContext: 'full-snapshot' as SerializationContext,
+        shouldSerializeScroll: true,
+      },
+      {
+        description: 'does not serialize scroll position during mutation',
+        serializationContext: 'mutation' as SerializationContext,
+        shouldSerializeScroll: false,
+      },
+    ].forEach(({ description, serializationContext, shouldSerializeScroll }) => {
+      it(description, () => {
+        const element = document.createElement('div')
+        Object.assign(element.style, { width: '100px', height: '100px', overflow: 'scroll' })
+        const inner = document.createElement('div')
+        Object.assign(inner.style, { width: '200px', height: '200px' })
+        element.appendChild(inner)
+        sandbox.appendChild(element)
+        element.scrollBy(10, 20)
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader’s attention to comments requiring more involvement.  -->
**🥜: nitpick:** would be nice to have this run in a `beforeEach` block

Reply:
What benefit do you see in extracting that to a `beforeEach` statement?

Reply:
On second thought, I find these lines quite useful to understand the context of each test. Feel free to ignore